### PR TITLE
fix: SSM prefix

### PR DESCRIPTION
**Pull request by npalm**
SSM prefix not applied for path. 

- Use prefix for all SSM paramaters
- Moved prefix to path instead of name for mult-runner
- Add optionals for ssm path variable

### CODE REVIEW

File: modules/multi-runner/variables.tf [hunk 1]
```diff
@@ -515,13 +515,9 @@ variable "workflow_job_queue_configuration" {
...
+    root    = optional(string, "github-action-runners")
+    app     = optional(string, "app")
+    runners = optional(string, "runners")
   })
-  default = {
-    root    = "github-action-runners"
-    runners = "runners"
-    app     = "app"
-  }
+  default = {}
```
Comment (GuptaNavdeep1983) [inline modules/multi-runner/variables.tf:522]: I am not sure what is the use of this?

File: variables.tf [hunk 2]
```diff
@@ -756,15 +756,10 @@ variable "enable_user_data_debug_logging_runner" {
...
+    runners    = optional(string, "runners")
+    use_prefix = optional(bool, true)
   })
-  default = {
-    root       = "github-action-runners"
-    use_prefix = true
-    runners    = "runners"
-    app        = "app"
-  }
+  default = {}
```
Comment (GuptaNavdeep1983) [inline variables.tf:764]: Same as above.

File: modules/multi-runner/variables.tf (hunk 1 above)
Comment (npalm) [reply modules/multi-runner/variables.tf:522]: Is not used, leaving the default let it fail.

Comment (npalm) [reply modules/multi-runner/variables.tf:522]: With this example you can try it your self,

```ncl
variable "test" {
  type = object({
    name = optional(string, "jonh")
    age  = optional(number, 20)
  })
  default = {}
}

output "test" {
  value = var.test
}
```

By setting the default to `{}` the object get a required value, the optionals ensures if object values are not set, they set with the default.

**Review by GuptaNavdeep1983 — APPROVED** [review]

**Pull request merged**